modules/cluster-logging-collector-tolerations.adoc

```diff
@@ -1,65 +1,79 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-collector.adoc
+// * logging/scheduling_resources/logging-taints-tolerations.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-collector-tolerations_{context}"]
 = Using tolerations to control the log collector pod placement
 
-You can ensure which nodes the logging collector pods run on and prevent
-other workloads from using those nodes by using tolerations on the pods.
-
-You apply tolerations to logging collector pods through the `ClusterLogging` custom resource (CR)
-and apply taints to a node through the node specification. You can use taints and tolerations
-to ensure the pod does not get evicted for things like memory and CPU issues.
-
 By default, the logging collector pods have the following toleration:
 
 [source,yaml]
 ----
-tolerations:
-- key: "node-role.kubernetes.io/master"
-  operator: "Exists"
-  effect: "NoExecute"
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+# ...
+spec:
+# ...
+  collection:
+    logs:
+      type: vector
+      vector:
+        tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoExecute
+# ...
 ----
 
 .Prerequisites
 
-* The Red Hat OpenShift Logging and Elasticsearch Operators must be installed.
+* You have installed the {clo} and {oc-first}.
 
 .Procedure
 
-. Use the following command to add a taint to a node where you want logging collector pods to schedule logging collector pods:
+. Add a taint to a node where you want logging collector pods to schedule logging collector pods, by running the following command:
 +
 [source,terminal]
 ----
-$ oc adm taint nodes <node-name> <key>=<value>:<effect>
+$ oc adm taint nodes <node_name> <key>=<value>:<effect>
 ----
 +
-For example:
-+
+.Example command
 [source,terminal]
 ----
 $ oc adm taint nodes node1 collector=node:NoExecute
 ----
 +
-This example places a taint on `node1` that has key `collector`, value `node`, and taint effect `NoExecute`.
-You must use the `NoExecute` taint effect. `NoExecute` schedules only pods that match the taint and removes existing pods
-that do not match.
+This example places a taint on `node1` that has key `collector`, value `node`, and taint effect `NoExecute`. You must use the `NoExecute` taint effect. `NoExecute` schedules only pods that match the taint and removes existing pods that do not match.
```

Comment:
I know we are just moving around this statement but I don't know this is accurate:

... and removes existing pods that do not match.

I do not imagine users make modifications to their workloads to ensure their pods get scheduled along side the collector.  There is something fishy about this statement

Reply:
I'd prefer to leave this as is for now to avoid increasing the scope of the PR too much beyond restructuring.